sklearn/gaussian_process/kernels.py

```diff
@@ -2131,6 +2131,20 @@ class PairwiseKernel(Kernel):
         All entries of this dict (if any) are passed as keyword arguments to
         the pairwise kernel function.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_friedman2
+    >>> from sklearn.gaussian_process import GaussianProcessRegressor
+    >>> from sklearn.gaussian_process.kernels import PairwiseKernel
+
+    >>> X, y = make_friedman2(n_samples=500, noise=0, random_state=0)
+    >>> kernel = PairwiseKernel(metric='rbf')
+    >>> gpr = GaussianProcessRegressor(kernel=kernel,
+    ...         random_state=0).fit(X, y)
```

Comment:
```suggestion
    >>> gpr = GaussianProcessRegressor(kernel=kernel, random_state=0)
    >>> gpr.fit(X, y)
```

Reply:
I'd rather not split this, because inlining the `fit` is more aligned with the other examples in this file. Splitting off the `gpr.fit` would then also require another output line everywhere (to be consistent) and in some of the other examples the `__repr__` for `ConstantKernel(2)` does not look so nice (`1.41**2` instead of `2`).